src/settings.rs

```diff
@@ -58,6 +62,8 @@ pub struct Settings {
     /// Fields displayed in the returned documents
     #[serde(skip_serializing_if = "Option::is_none")]
     pub displayed_attributes: Option<Vec<String>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
```

A comment is missing here